spec/item_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'pry'
+require './lib/item'
+require 'bigdecimal'
+
+RSpec.describe do
+  
+  it 'exists' do
+    item = Item.new({
+      :id          => 1,
+      :name        => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price  => BigDecimal(10.99,4),
+      :created_at  => Time.now,
+      :updated_at  => Time.now,
+      :merchant_id => 2
+    })
+
+    expect(item).to be_a(Item)
+  end
+
+  it 'has an id number' do
+    item = Item.new({
+      :id          => 1,
+      :name        => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price  => BigDecimal(10.99,4),
+      :created_at  => Time.now,
+      :updated_at  => Time.now,
+      :merchant_id => 2
+    })
+
+    expect(item.id).to eq(1)
+  end
+
+  it 'has a name' do
+    item = Item.new({
+      :id          => 1,
+      :name        => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price  => BigDecimal(10.99,4),
+      :created_at  => Time.now,
+      :updated_at  => Time.now,
+      :merchant_id => 2
+    })
+
+    expect(item.name).to eq('Pencil')
+  end
+
+  it 'has a description' do
+    item = Item.new({
+      :id          => 1,
+      :name        => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price  => BigDecimal(10.99,4),
+      :created_at  => Time.now,
+      :updated_at  => Time.now,
+      :merchant_id => 2
+    })
+
+    expect(item.description).to eq("You can use it to write things")
+  end
+
+  it 'has a unit price' do
+    item = Item.new({
+      :id          => 1,
+      :name        => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price  => BigDecimal(10.99,4),
+      :created_at  => Time.now,
+      :updated_at  => Time.now,
+      :merchant_id => 2
+    })
+
+    expect(item.unit_price).to eq(BigDecimal(10.99,4))
+  end
+
+  it 'creates a Time instance when it is created' do
+    item = Item.new({
+      :id          => 1,
+      :name        => "Pencil",
+      :description => "You can use it to write things",
+      :unit_price  => BigDecimal(10.99,4),
+      :created_at  => Time.now,
+      :updated_at  => Time.now,
+      :merchant_id => 2
+    })
+
+    expect(item.created_at).to be_a(Time)
```

Comment:
we will need to do some integration testing to make sure it's logging a time when the item is created